template-base-2/template/src/components/common/map/GoogleMapsComponent.js

```diff
@@ -1,5 +1,5 @@
 import { useSelector } from 'react-redux';
-import 'mapbox-gl/dist/mapbox-gl.css';
+import 'maplibre-gl/dist/maplibre-gl.css';
```

Comment:
I don't remember why it's here, but I think this could just be removed in all `GoogleMapsComponent` files.

Reply:
I think so, I'm going to remove it and just keep the import in Map.js